@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -1190,22 +1190,21 @@ export class VisibleTileSet {
             }
         });
 
-        // If elevation is to be taken into account create extended frustum:
+        // If elevation is to be taken into account extend view frustum:
         // (near ~0, far: maxVisibilityRange) that allows to consider tiles that
         // are far below ground plane and high enough to intersect the frustum.
         if (elevationRangeSource !== undefined) {
-            const fp = MapViewUtils.getCameraFrustumPlanes(this.m_frustumIntersection.camera);
-            fp.near = this.m_viewRange.minimum;
-            fp.far = this.m_viewRange.maximum;
-            this.m_projectionMatrixOverride.makePerspective(
```

Comment:
why did you remove makePerspective? I don't see why you need to use a PerspectiveCamera, when makePerspective should be all you need (it seems that you just need to fix the near / far planes)?

Reply:
Newest version of THREE.js uses `top, bottom, left, right, near, far` as parameters for `Matrix4.makePerspective()`. Calculating these parameters from `fov, aspect, near, far` is encapsuled in the `PerspectiveCamera`. I didn't want to reimplement the code to calculate these parameters in harp.gl as THREE.js may change the way it creates the `projectionMatrix` from the camera parameters or changes the API again.

Reply:
I understand, I made the same mistake as the author of this code, thinking that you can change the near / far without adjusting the sides. My bad.